ADR/adr-0010-namespace-metadata

```diff
@@ -0,0 +1,39 @@
+# ADR-0010 Namespace Metadata
+
+Date: 2022-December-9
+
+## Status
+
+Accepted
+
+## Approvers
+
+* Ann Marie Fred
+* Gorkem Ercan
+
+## Reviewers
+
+## Context
+
+We need metadata on our namespaces to make Stone Soup easier to operate and maintain. Standardizing namespace metadata will make it easier for us to search our logs and metrics across clusters. It will also allow us to enable logging for outgoing network traffic, one of our security requirements.
+
+## Namespace labels
+
+We will apply the following labels to Stone Soup namespaces, to make them easier to find.
+
+- `stonesoup.redhat.com/namespacetype: "controller"` for namespaces containing Stone Soup controllers
```

Comment:
could you please define what you mean by "StoneSoup controller"? Do count also controllers of the OpenShift Pipeline operator (Tekton)? How about ArgoCD and Sandbox controllers?

Reply:
I think it means tekton, yes. I think it does _not_ mean the `openshift-gitops` namespace, but it does mean the `gitops-service-argocd` namespace.

Reply:
I added additional details.